.github/workflows/nm-upload-assets-to-gcp.yml

```diff
@@ -68,6 +84,16 @@ jobs:
               with:
                   path: assets
 
+            - name: push wheel to pypi.org
+              # this workflow is only run if push-to-pypi is True, and we only
+              # want to push RELEASE wheels to the external pypi.org
+              if: ${{ inputs.wf_category }} == "RELEASE"
```

Comment:
I think this condition needs to be: ${{ inputs.wf_category  == "RELEASE" }}

Reply:
yeah ... or you can also use "contains" ... but i like the simple one better